packages/ia-components/package.json

```diff
@@ -8,13 +8,15 @@
         "storybook:start": "start-storybook -p 9001 -c .storybook",
         "test:generate-output": "mkdir jest-test-utils && jest --json --outputFile=jest-test-utils/jest-test-results.json || true",
         "test:update-snapshots": "jest --updateSnapshot",
-        "test": "jest --watch --coverage"
+        "test": "jest --watch --coverage",
+        "test:integration": "node_modules/.bin/jest --config jest-test-utils/jest.config.js"
```

Comment:
QUESTION: Where is the `jest-test-utils/jest.config.js` file located?

Reply:
Still on my local file system because the directory is in .gitignore. Moved them to the integration directory.